docs/pages/dev.md

```diff
@@ -12,20 +12,15 @@ layout: default
 
 # Virtual environments
 
-| Name                                                                  | Short description                                                                   | 🚦  | Used by |
-| --------------------------------------------------------------------- | ----------------------------------------------------------------------------------- | --- | :-----: |
-| [mamba](https://mamba.readthedocs.io/en/latest/user_guide/mamba.html) | A tool to manage conda environments. Faster than conda for installing dependencies. | 🟢  |         |
-| [conda](https://conda.io/projects/conda/en/latest/)                   | Conda quickly installs, runs, and updates packages and their dependencies.          | 🟠  |         |
-| [pipenv](https://pipenv.pypa.io/en/latest/)                           | A tool that automatically creates and manages a virtualenv for your projects.       | 🟠  |         |
-| [venv](https://docs.python.org/3/library/venv.html)                   | A tool to create isolated Python environments.                                      | 🔴  |         |
-| [virtualenv](https://virtualenv.pypa.io/en/latest/)                   | A tool to create isolated Python environments which offers more features than venv. | 🟠  |         |
-| [pyenv](https://github.com/pyenv/pyenv)                               | A tool that lets you easily switch between multiple versions of Python.             | 🟠  |         |
+| Name                                                | Short description                                                                    | 🚦  | Used by |
+| --------------------------------------------------- | ------------------------------------------------------------------------------------ | --- | :-----: |
+| [conda](https://conda.io/projects/conda/en/latest/) | Conda quickly installs, runs, and updates packages and their dependencies.           | 🟢  |         |
```

Comment:
```suggestion
| [conda](https://docs.conda.io/projects/conda/en/stable/) | Conda quickly installs, runs, and updates packages and their dependencies.           | 🟢  |         |
```

This is what https://docs.conda.io/projects/conda redirects to, and I think better to link to the stable version of the docs (instead of latest which I assume is for unreleased versions)